### PR TITLE
[DEV-141] Actor info 1

### PR DESCRIPTION
**Pull request by aminnoah**
[DEV-141](https://artistguide.atlassian.net/browse/DEV-141)

**Description:**

DEV-141 Actor Info 1. Initial layout and styles. Need to make certain input fields as conditionals to others as well as determine best practices for implementing generic components (ex: Dropdown.tsx doesn't not like when I use it. Similar to RadioButton.tsx)

**Screenshots (if applicable):**

Screenshots here

**PR Checklist:**

- [ ] If you haven't made changes to `package.json`, `package-lock.json` is not present in this PR
- [ ] You have tested these changes locally by running the app and letting it build successfully
- [ ] Eslint ran successfully upon commit and `--force` was not used to force this PR

_Make sure you **squash and merge** rather than simply merging all commits at once_.

At least **1** PR approval is required by a Tech Lead to be merged. **2** approvals total are required.

### CODE REVIEW

**Review by bestfoodalex — COMMENTED** [review]
Don't commit package-lock.json if you haven't updated package.json. Run:
`git checkout master package-lock.json`

File: src/components/SignUp/ActorInfo1.tsx [hunk 1]
```diff
@@ -1,22 +1,222 @@
...
+
+    const target = {
+      name: 'actorInfo1Ethnicities',
+      value: newEthnicities
+    };
+
+    setForm({ target });
+  };
+
+  // const genderRoleChange = (checkValue: any, role: string) => {
```
Comment (bestfoodalex) [inline src/components/SignUp/ActorInfo1.tsx:93]: Don't leave commented out code in

File: src/components/SignUp/ActorInfo1.tsx [hunk 2]
```diff
@@ -32,4 +232,12 @@ const ImageCol = styled(Col)`
   max-width: 100%;
 `;
 
+const CAGLabel = styled(Form.Label)`
+  color: ${colors.mainFont};
+  font-family: ${fonts.mainFont};
+  font-size: 20px;
+`;
+
+const InnerEthnicities = styled(Checkbox)``;
```
Comment (bestfoodalex) [inline src/components/SignUp/ActorInfo1.tsx:241]: No need for this styled component if you don't have any styles for it

**Comment by bestfoodalex** [comment]
Do you mind adding a screenshot to this PR of what the page looks like, as well?

**Comment by aminnoah** [comment]
Fixed the style edits and the pronouns input field is now dynamic if the value is set as other.

File: src/components/SignUp/ActorInfo1.tsx [hunk 3]
```diff
@@ -1,22 +1,203 @@
...
+                />
+              </Form.Group>
+              <Form.Group>
+                <CAGLabel>Do you identify as any of the following:</CAGLabel>
+                <PrivacyPar>
+                  Note: This list is not exhaustive nor hyper-specific. The goal
+                  of selecting your identity is to help cast roles that call for
+                  a certain demographic. Please select as may options as you
+                  feel fits your identity. More info on our process{' '}
+                  <Link to="">here</Link>.
```
Comment (bestfoodalex) [inline src/components/SignUp/ActorInfo1.tsx:168]: Use `#` for the to field for now until we have somewhere to go: `<Link to="#">here</Link>`

File: src/components/SignUp/ActorInfo1.tsx [hunk 4]
```diff
@@ -32,4 +213,30 @@ const ImageCol = styled(Col)`
...
+`;
+
+const PrivacyPar = styled.p`
+  color: ${colors.mainFont};
+  font-family: ${fonts.lora};
+  font-size: 18px;
+  letter-spacing: 0px;
+  margin-top: 17px;
+
+  link {
```
Comment (bestfoodalex) [inline src/components/SignUp/ActorInfo1.tsx:235]: `link` isn't the underlying tag, `a` is. If you also don't need to change the color of it, you don't need to have the style block here for it at all.

Comment (aminnoah) [reply src/components/SignUp/ActorInfo1.tsx:235]: Makes sense. The flow has the link being red, but I just thought for consistency I should keep it the same color. What do you think?

Comment (bestfoodalex) [reply src/components/SignUp/ActorInfo1.tsx:235]: @aminnoah yup! That sounds good. So you can remove this link {} block altogether then 👍🏻

**Review by bestfoodalex — APPROVED** [review]
Looks good

**Pull request merged**